Smart4SURE/BridgeInfo.plist

```diff
@@ -2,6 +2,33 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSAppTransportSecurity</key>
```

Comment:
By default, since iOS 9, iOS SDK requires all URLs accessed by apps to be https://. Since not all sites support https://, they provided this as a workaround to allow exceptions for specific URLs or domains until those third-party sites you depend on update to support https://.